Mathlib/Algebra/Order/Group/Basic.lean

```diff
@@ -34,6 +34,13 @@ lemma zpow_right_strictMono (ha : 1 < a) : StrictMono fun n : ℤ ↦ a ^ n := f
     _ < a ^ m * a ^ (n - m) := mul_lt_mul_left' (one_lt_zpow' ha <| Int.sub_pos_of_lt h) _
     _ = a ^ n := by simp [← zpow_add, m.add_comm]
 
+@[to_additive zsmul_strictAnti_left]
+lemma zpow_right_strictAnti (ha : a < 1) : StrictAnti fun n : ℤ ↦ a ^ n := fun m n h ↦ calc
```

Comment:
I think then we should also have `zpow_lt_one'` above. The `to_additive` name might have to be `zsmul_neg'` due to the clash with `zsmul` of `neg` powers.

Reply:
It seems it would be useful in your proof without to rely on `one_lt_inv'` (?)

Reply:
There are 4 natural lemmas here, we have 1 of them. Possibly, we should have all 4, but IMHO this is out of scope of this PR.